app/server.ts

```diff
@@ -0,0 +1,41 @@
+import * as Express from 'express';
+import * as BodyParser from 'body-parser';
+import * as Morgan from 'morgan';
+
+import { PORT } from './modules/util/config';
+import { BasePath as ProjectBasePath, projectRouter } from './modules/project';
+
+export class Server {
+    server: Express.Application;
+    port: string;
+
+    constructor() {
+        this.server = Express();
+        this.port = PORT;
+        this.config();
+        this.defaultRoute();
+    }
+
+    config = () => {
+        this.server.use(Morgan('combined'));
+        this.server.use(BodyParser.urlencoded({ extended: false }));
+        this.server.use(BodyParser.json());
+    };
+
+    defaultRoute = () => {
```

Comment:
This shouldn't exist here. Ideally if you want to have some default routes ( generally you create health routes ) you can put them as a separate module somewhere else.

- [ ] Create a health module as a separate unit